app/styleguide/page.tsx

```diff
@@ -329,17 +341,29 @@ function Page() {
 
   const [editorContent, setEditorContent] = useState(richtextDefault);
 
+  const [sidebarDirection, setSidebarDirection] = useState('right');
+  const toggleSidebarDirection = () => {
+    if (sidebarDirection == 'right') setSidebarDirection('left');
+    else setSidebarDirection('right');
+  }
+
+  const [sidebarOpen, setSidebarOpen] = useState(true);
+  const toggleSidebar = () => {
+    setSidebarOpen(!sidebarOpen);
+  }
+
   return (
     <>
-      <h1>Living Styleguide</h1>
+      <LayoutWithSidebar id="sgLayout" sidebarPosition="left">
+        <SidebarContainer isOpen={true}>
+          <h3>Contents</h3>
 
-      <div id="sgLayout">
-        <div id="sgNav">
           <a href="#_intro">Introduction</a>
           <a href="#_brand">Branding & Colours</a>
           <a href="#_typography">Typography</a>
           <a href="#_icons">Icons</a>
           <a href="#_layout">Layout</a>
+          <a href="#_containers">Containers</a>
```

Comment:
L366: Remember to add "id=_containers" to the Containers section so that this links to something